eth/protocols/eth/handler.go

```diff
@@ -180,12 +180,12 @@ var eth66 = map[uint64]msgHandler{
 	BlockHeadersMsg:               handleBlockHeaders66,
 	GetBlockBodiesMsg:             handleGetBlockBodies66,
 	BlockBodiesMsg:                handleBlockBodies66,
-	GetNodeDataMsg:                handleGetNodeData66,
-	NodeDataMsg:                   handleNodeData66,
-	GetReceiptsMsg:                handleGetReceipts66,
-	ReceiptsMsg:                   handleReceipts66,
-	GetPooledTransactionsMsg:      handleGetPooledTransactions66,
-	PooledTransactionsMsg:         handlePooledTransactions66,
+	//GetNodeDataMsg:                handleGetNodeData66,
+	//NodeDataMsg:                   handleNodeData66,
```

Comment:
It seems that we will disconnect nodes who send these, is that intentional?

Reply:
yes. since we use our own trie design, we cannot support snapshot/snapshot yet.